app/serializers/location_serializer.rb

```diff
@@ -3,14 +3,18 @@
 class LocationSerializer
   include JSONAPI::Serializer
 
+  INCLUDED_ATTRIBUTES = %i[
```

Comment:
You can avoid all of this noise by inspecting the `attributes_to_serialize` class method of each serializer - and I think it's much more idiomatic to do this rather than define an array of attributes in every serializer. Plus this will also include attributes that are defined individually which we do in a few places. See:

```ruby
[1] pry(main)> LocationSerializer.attributes_to_serialize
=> {:key=>#<FastJsonapi::Attribute:0x00007fdc9692f868 @conditional_proc=nil, @key=:key, @method=:key>,
 :title=>#<FastJsonapi::Attribute:0x00007fdc9692f818 @conditional_proc=nil, @key=:title, @method=:title>,
 :location_type=>#<FastJsonapi::Attribute:0x00007fdc9692f7c8 @conditional_proc=nil, @key=:location_type, @method=:location_type>,
 :nomis_agency_id=>#<FastJsonapi::Attribute:0x00007fdc9692f778 @conditional_proc=nil, @key=:nomis_agency_id, @method=:nomis_agency_id>,
 :can_upload_documents=>#<FastJsonapi::Attribute:0x00007fdc9692f728 @conditional_proc=nil, @key=:can_upload_documents, @method=:can_upload_documents>,
 :disabled_at=>#<FastJsonapi::Attribute:0x00007fdc9692f6d8 @conditional_proc=nil, @key=:disabled_at, @method=:disabled_at>}
[2] pry(main)> LocationSerializer.attributes_to_serialize.keys
=> [:key, :title, :location_type, :nomis_agency_id, :can_upload_documents, :disabled_at]
```